pom.xml

```diff
@@ -822,7 +822,7 @@
     <maven.min.version>3.5.0</maven.min.version>
     <java.min.version>${compileSource}</java.min.version>
     <!-- Dependencies -->
-    <hadoop-three.version>3.3.5</hadoop-three.version>
+    <hadoop-three.version>3.4.0</hadoop-three.version>
```

Comment:
netty3 is removed from hadoop 3.4.0.
It's a shame we didn't mention it in the Hadoop 3.4.0 release note.

Reply:
IMO we should provide a way to build hbase without netty 3 if we are building hbase with hadoop 3.4.0+

Reply:
HBase only depends on Netty 3 transitively via Hadoop.
Building with Hadoop 3.4.0 automagically removes Netty 3 dependencies.